memcached.c

```diff
@@ -1427,21 +1432,11 @@ static void einfo_set_ascii_tail_string(eitem_info *einfo)
 static int build_udp_headers(conn *c)
 {
     assert(c != NULL);
-    unsigned char *hdr;
-
-    if (c->msgused > c->hdrsize) {
-        void *new_hdrbuf;
-        if (c->hdrbuf)
-            new_hdrbuf = realloc(c->hdrbuf, c->msgused * 2 * UDP_HEADER_SIZE);
-        else
-            new_hdrbuf = malloc(c->msgused * 2 * UDP_HEADER_SIZE);
-        if (! new_hdrbuf)
-            return -1;
-        c->hdrbuf = (unsigned char *)new_hdrbuf;
-        c->hdrsize = c->msgused * 2;
-    }
+    unsigned char *hdr = realloc(c->hdrbuf, c->msgused * UDP_HEADER_SIZE);
+    if(! hdr) return -1;
```

Comment:
```suggestion
    if (!hdr) return -1;
```
두 방식이 혼용 중인데 전자의 비율이 높으므로 붙여 쓰는 형태로 통일하는 게 좋을 것 같습니다.
- arcus-memcached
  - `if (!value)`: 122개
  - `if (! value)`: 56개
- arcus-memcached-ee
  - `if (!value)`: 193개
  - `if (! value)`: 60개

참고로 `if(`와 `if (` 개수는 아래와 같습니다.
- arcus-memcached
  - `if(`: 22개
  - `if (`: 4972개
- arcus-memcached-ee
  - `if(`: 22개
  - `if (`: 10896개